extensions/sql-migration/src/wizard/loginMigrationTargetSelectionPage.ts

```diff
@@ -744,6 +745,29 @@ export class LoginMigrationTargetSelectionPage extends MigrationWizardPage {
 							const selectedVm = this.migrationStateModel._targetSqlVirtualMachines.find(vm => vm.name === value);
 							if (selectedVm) {
 								this.migrationStateModel._targetServerInstance = utils.deepClone(selectedVm)! as SqlVMServer;
+								this.migrationStateModel._vmInstanceView = await getVMInstanceView(this.migrationStateModel._targetServerInstance, this.migrationStateModel._azureAccount, this.migrationStateModel._targetSubscription);
+								this.migrationStateModel._targetServerInstance.networkInterfaces = await NetworkInterfaceModel.getVmNetworkInterfaces(
+									this.migrationStateModel._azureAccount,
+									this.migrationStateModel._targetSubscription,
+									this.migrationStateModel._targetServerInstance);
+
+								this.wizard.message = { text: '' };
+
+								// validate power state from VM instance view
+								if (!this.migrationStateModel._vmInstanceView.statuses.some(status => status.code.toLowerCase() === 'PowerState/running'.toLowerCase())) {
```

Comment:
need to make 'PowerState/running' as constant?

Reply:
good idea